content/docs/introduction/media.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Media
+sort_rank: 7
+---
+
+# Media
+
+Resources on the Internet helpful to get started with Prometheus.
+
+## Blogs
+
+* This site has its own [blog](http://prometheus.io/blog/).
+* [SoundCloud's blog post announcing Prometheus](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud) – a more elaborate overview than the one given on this site.
+* The [monitoring series](http://www.boxever.com/tag/monitoring) on Boxever's tech blog.
+
+## Recorded talks
+
+* [Prometheus: A Next-Generation Monitoring System](https://www.usenix.org/conference/srecon15europe/program/presentation/rabenstein) – Julius Volz and Björn Rabenstein at SREcon15 Europe, Dublin.
+* [What is your application doing right now?](youtu.be/Z0LlilNpX1U) – Matthias Gruter, Transmode, at DevOps Stockholm Meetup.
```

Comment:
:+1: for keeping the scheme - wouldn't this just be a relative link otherwise anyways?